canonical-web-and-design/jaas-dashboard
# Use ghost bundle image in the model details

## Done

Use ghost bundle image in the model details

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- View one of your model details you should see the ghost bundle image.

## Screenshots

<img width="360" alt="Screen Shot 2019-11-07 at 2 00 16 PM" src="https://user-images.githubusercontent.com/532033/68423201-3fb04f80-0167-11ea-8950-87d830fdf3ae.png">
